modules/monitoring/manifests/network_checks.pp

```diff
@@ -0,0 +1,22 @@
+# == Class: monitoring::network_checks
+#
+# Sets up vDC gateways as separate hosts so they can become parents of child
+# hosts dependent on the state of their gateway.
+#
```

Comment:
@mattbostock, I think the description should be updated, I set it as "network_checks" in the (unlikely) event there were other network devices we may want to monitor in the future.